rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -536,6 +553,145 @@ void saveOldFederation_after_RSKIP123_should_save_null() {
         );
     }
 
+    @Test
+    void getNewFederationBtcUTXOs_before_RSKIP284_before_RSKIP293_testnet() throws IOException {
+        testGetNewFederationBtcUTXOs(false, false, NetworkParameters.ID_TESTNET);
+    }
+
+    @Test
+    void getNewFederationBtcUTXOs_before_RSKIP284_before_RSKIP293_mainnet() throws IOException {
+        testGetNewFederationBtcUTXOs(false, false, NetworkParameters.ID_MAINNET);
+    }
+
+    @Test
+    void getNewFederationBtcUTXOs_before_RSKIP284_after_RSKIP293_testnet() throws IOException {
+        testGetNewFederationBtcUTXOs(false, true, NetworkParameters.ID_TESTNET);
+    }
+
+    @Test
+    void getNewFederationBtcUTXOs_before_RSKIP284_after_RSKIP293_mainnet() throws IOException {
+        testGetNewFederationBtcUTXOs(false, true, NetworkParameters.ID_MAINNET);
+    }
+
+    @Test
+    void getNewFederationBtcUTXOs_after_RSKIP284_before_RSKIP293_testnet() throws IOException {
+        testGetNewFederationBtcUTXOs(true, false, NetworkParameters.ID_TESTNET);
+    }
+
+    @Test
+    void getNewFederationBtcUTXOs_after_RSKIP284_before_RSKIP293_mainnet() throws IOException {
+        testGetNewFederationBtcUTXOs(true, false, NetworkParameters.ID_MAINNET);
+    }
+
+    @Test
+    void getNewFederationBtcUTXOs_after_RSKIP284_after_RSKIP293_testnet() throws IOException {
+        testGetNewFederationBtcUTXOs(true, true, NetworkParameters.ID_TESTNET);
+    }
+
+    @Test
+    void getNewFederationBtcUTXOs_after_RSKIP284_after_RSKIP293_mainnet() throws IOException {
+        testGetNewFederationBtcUTXOs(true, true, NetworkParameters.ID_MAINNET);
+    }
```

Comment:
shouldn't we make these a parameterized test?

Reply:
I agree

Reply:
Updated.